docs/setupstoragepools.md

```diff
@@ -86,8 +86,7 @@ spec:
 
 cStor provides storage scalability along with ease of deployment and usage. cStor can handle multiple disks of same size per Node and create different storage pools. These Storage Pools can be used to create cStor volumes which can be utilized to run applications.
```

Comment:
Delete space before cStor
These Storage Pools... = You can use these storage pools to create cStor volumes which you can utilize to run applications.